core/src/main/java/novoda/lib/sqliteprovider/sqlite/IDatabaseMetaInfo.java

```diff
@@ -8,9 +8,7 @@
 
 public interface IDatabaseMetaInfo {
 
-
-
-    public enum SQLiteType {
```

Comment:
Why does this call for a major release? The `public` modifier was redundant, since we are in an interface, everything is public already 😛